changelog/unreleased/update-web-1.0.1.md

```diff
@@ -0,0 +1,8 @@
+Change: Update ownCloud Web to v1.0.1
+
+Tags: web
+
+We updated ownCloud Web to v1.0.1. Please refer to the changelog (linked) for details on the web release.
+
+https://github.com/owncloud/ocis/pull/1191
+https://github.com/owncloud/phoenix/releases/tag/v1.0.1
```

Comment:
```suggestion
https://github.com/owncloud/web/releases/tag/v1.0.1
```

How did this end up pointing to the old repo?
The link redirects to `owncloud/web` so it is not broken.